sdk/python/packages/flet/src/flet/core/page.py

```diff
@@ -930,17 +930,19 @@ def run_task(
         handler: Callable[InputT, Awaitable[RetT]],
         *args: InputT.args,
         **kwargs: InputT.kwargs,
-    ) -> asyncio.Future[RetT]:
+    ) -> Future[RetT]:
         _session_page.set(self)
         assert asyncio.iscoroutinefunction(handler)
 
         future = asyncio.run_coroutine_threadsafe(handler(*args, **kwargs), self.__loop)
 
         def _on_completion(f):
-            exception = f.exception()
-
-            if exception:
-                raise exception
+            try:
+                exception = f.exception()
+                if exception:
+                    raise exception
+            except CancelledError:
+                pass
```

Comment:
Should we be [logging](https://github.com/search?q=repo%3Aflet-dev%2Fflet%20logging&type=code) this exception?

Reply:
I think this doesn't need to be logged since the only way this exception raises is when we cancelling task manually and this operation is safe as we can do it only when `await asyncio.sleep` running inside task